packages/components/src/theme/ComposerTheme/Components/TextPairing.ts

```diff
@@ -0,0 +1,223 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const TextPairing: ComponentStyleConfig = {
+  parts: ['label', 'subLabel'],
+  variants: {
+    'typepairing-font-header-4XL': {
```

Comment:
Can we rename typepairing to textpairing if design agrees?

Reply:
ask them 😉

Reply:
done!